mk/dev.mk

```diff
@@ -1,7 +1,9 @@
 KUMA_DIR ?= .
 ENVOY_VERSION = $(word 5, $(shell ${KUMA_DIR}/tools/releases/version.sh))
+CI_TOOLS_VERSION = $(word 6, $(shell ${KUMA_DIR}/tools/releases/version.sh))
+KUMA_CHARTS_URL = https://kumahq.github.io/charts
 
-CI_TOOLS_DIR ?= ${HOME}/.kuma-dev
+CI_TOOLS_DIR ?= ${HOME}/.kuma-dev/${CI_TOOLS_VERSION}
```

Comment:
Isn't it just: `PULP_DIST_NAME ?= $(DISTRIBUTION_NAME)-$(shell echo $(BUILD_INFO_VERSION) | awk -F '.' '{ print $$1"."$$2"."$$3 }')` ?

Pulled from my WIP: https://github.com/kumahq/kuma/pull/6137/files

Reply:
I think it's not fully this. I am taking the information from `describedTag` the closest tag that I can get to. Maybe I could switch to using it but still, it requires some customs changes. 
E.g.:

When you are on a `master` branch, the version is 0.0.0-preview, and the closest tag is 2.0.0
When you are on a `release-2.1` branch, the version is 2.1.1, but when you create a branch from release-2.1, it is going to return 0.0.0-preview. That makes it more complicated. I found the closest tag to be more reliable. I could maybe take the 2nd parameter from the `build_info` but still it requires parsing so we won't make different tools dirs for different branch/change in code

Reply:
arg indeed. that seems a little fragile but might work...

Reply:
I feel like adding `DISTRIBUTION_NAME` parameter might be a good option. We can keep different tooling depending on the project. Should I add it here and later in your PR just to remove the duplicate parameter?

Reply:
You can add it now (I still need some work on the linked PR). Though maybe `PROJECT_NAME` is better.